tests/std/tests/VSO_0000000_instantiate_containers/test.cpp

```diff
@@ -46,6 +46,20 @@ using namespace std;
 
 int main() {} // COMPILE-ONLY
 
+// FUNCTION TEMPLATE For_each_tuple_element
+template <class Tpl, class Fx, size_t... Indices>
+void for_each_tuple_element_impl(Tpl&& Tuple, Fx Func, index_sequence<Indices...>) {
+    // call Func() on the _Indices elements of _Tuple
+    int ignored[] = {(static_cast<void>(Func(get<Indices>(forward<Tpl>(Tuple)))), 0)...};
+    (void) ignored;
+}
+
+template <class Tpl, class Fx>
+void for_each_tuple_element(Tpl&& Tuple, Fx Func) { // call Func() on each element in _Tuple
+    for_each_tuple_element_impl(
+        forward<Tpl>(Tuple), Func, make_index_sequence<tuple_size_v<remove_reference_t<Tpl>>>());

```

Comment:
Cleanup of pre-existing code: nowadays, we try to use `{}` to construct temporary objects because `()` looks like a function call. Here, `make_index_sequence` is an alias template, not a function.

```suggestion
        forward<Tpl>(Tuple), Func, make_index_sequence<tuple_size_v<remove_reference_t<Tpl>>>{});
```